content/docs/user-guide/experiment-management/running-experiments.md

```diff
@@ -192,11 +191,11 @@ gets interrupted (e.g. with `Ctrl+C`, or by an error). Without interruption, a
 "wrap-up" checkpoint will be added (if needed), so that changes to pipeline
 outputs don't remain in the workspace.
 
-Subsequent uses of `dvc exp run` will continue from the latest checkpoint (using
+Subsequent uses of `dvc exp run` will resume from the latest checkpoint (using
```

Comment:
👍🏼 

Should we also change  continue -> resume  in [here](https://dvc.org/doc/command-reference/exp/run#-r)?

Reply:
@jorgeorpinel  - I change the command reference as well. But please note I noticed the command references fell out of sync (at least partly) with the actual `dvc` command help messages, see [exp run -r](https://github.com/iterative/dvc/blob/main/dvc/commands/experiments/run.py#L69)
I guess this calls for a realignment pass(es) or some mechanism in place to avoid disparity

Reply:
Thanks!

> I noticed the command references fell out of sync (at least partly) with the actual `dvc` command help messages, see [exp run -r](https://github.com/iterative/dvc/blob/main/dvc/commands/experiments/run.py#L69)

Thanks for mentioning it.

> I guess this calls for a realignment pass(es) or some mechanism in place to avoid them falling out of sync

The core team has a manual check in their PRs where they decide when docs changes are required but being manual there's room for error of course, so yeah that happens! Cc @dberenbaum

Reply:
p.s. Actually, there's no un-alignment here. We don't intend to copy-paste option descriptions from help output. We often rewrite them completely in docs.

Reply:
p.p.s. if anything I'd say that the CLI output `Continue the specified checkpoint experiment` isn't as clear as the docs `continue an experiment from a specific checkpoint`. You don't need `--rev` to **resume** a checkpoint experiment, you need it to specify from which checkpoint to resume from.

Reply:
Created https://github.com/iterative/dvc/issues/7451 to follow up on that.

Reply:
> "We often rewrite them completely in docs."
Intriguing! 👀 
Understood